analytical/templatetags/google_analytics_gtag.py

```diff
@@ -72,6 +75,25 @@ def render(self, context):
             html = disable_html(html, 'Google Analytics')
         return html
 
+    def _get_event_commands(self, context):
+        values = (
+            context.get('google_analytics_event%s' % i) for i in range(1, 6)
+        )
+        params = [(i, v) for i, v in enumerate(values, 1) if v is not None]
+        commands = []
+        for _, var in params:
+            key = var[0]
+            value = var[1]
```

Comment:
Maybe this could be written as a single line, like this?

```suggestion
            key, value = var
```